src/pages/iou/MoneyRequestCategoryPage.js

```diff
@@ -51,7 +51,6 @@ function MoneyRequestCategoryPage({route, report}) {
                 title={translate('common.category')}
                 onBackButtonPress={navigateBack}
             />
-
```

Comment:
NAB: Let's remove the deletion of the line so that this file is otherwise untouched.